docs/docs/marketplace/marketplace_overview.md

```diff
@@ -0,0 +1,95 @@
+---
+id: marketplace-overview
+title: 'Marketplace : Overview'
+---
+
+# Marketplace : Overview
+
+With ToolJet Marketplace, ToolJet users can conveniently add custom plugins (datasources) to their workspaces. This feature enables users to create plugins that cater to their specific needs and integrate them seamlessly with ToolJet.
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/marketplace/overview/marketplace.png" alt="Marketplace" />
+
+</div>
+
+## Enabling the marketplace 
```

Comment:
"Marketplace" should be capitalised as it is a proper noun referring to the name of the platform.